Week_1_assignment.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
```

Comment:
The requirement is Read–Eval–Print–Loop, so the program should continuously read user input

Reply:
Advises are received with gratitude. I'll be mindful next time.
Thanks for spending time reviewing my code.